### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by KatatoniaSh** [WeAreMahsaAmini/Core#81] [Ideas]
If we wanna broaden our activities, we need to spend money on advertising, etc. So, it might be a good idea to have donations. Some trustable people like Hamed Esmaeilion would be a qualified person. I am not sure if it is possible to support people in Iran financially.

**Comment by AnonymousMaximus2** [comment]
The idea is great but

you need infrastructure for this idea,

and I explained this infrastructure here:

https://github.com/WeAreMahsaAmini/Core/discussions/115

**Comment by kaveh6** [comment]
It's a little bit more tricky to donate to people inside Iran . If you use crypto , they need to avoid selling it to official crypto exchange websites in Iran as it would be easy to track people and the usage of money .

**Comment by AnonymousMaximus2** [comment]
Crypto does not need to be sold, as Elon Musk said last year, just they can use Crypto as a medium of account,

for example in grocery when buying some tomatoes they can pay crypto against paying Rial !

**Comment by ehsaaanes** [comment]
Hi

Please check my idea here #163 and write a comment or let me know what you think about it.

**Reply by ghost** [reply]
We see your idea, please don't spam

**Comment by amniroh** [comment]
This is a great idea. Maybe creating a page in gofundme.com might be a great fit for this. Please also consider transferring a portion of the donations to trusted charity organizations (like child foundation and Golestan), since they may already be able to transfer the funds legally to Iranians.

**Reply by ehsaaanes** [reply]
Thank you for your reply, will send the app to those organizations to create their account. Please let me know if you would like to help building it. I’m in search for Ui/ us designer and backend developer.
